### PR TITLE
Add votca

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
```
2021-12-06T06:35:12.2947309Z -- The CXX compiler identification is GNU 9.4.0
2021-12-06T06:35:12.3074982Z -- Detecting CXX compiler ABI info
2021-12-06T06:35:12.4077142Z -- Detecting CXX compiler ABI info - done
2021-12-06T06:35:12.4258201Z -- Check for working CXX compiler: $BUILD_PREFIX/bin/x86_64-conda-linux-gnu-c++ - skipped
2021-12-06T06:35:12.4264592Z -- Detecting CXX compile features
2021-12-06T06:35:12.4275603Z -- Detecting CXX compile features - done
2021-12-06T06:35:12.4340554Z -- Performing Test COMPILER_SUPPORTS-Wall
2021-12-06T06:35:12.5305617Z -- Performing Test COMPILER_SUPPORTS-Wall - Success
2021-12-06T06:35:12.5317821Z -- Performing Test COMPILER_SUPPORTS-Wextra
2021-12-06T06:35:12.6236215Z -- Performing Test COMPILER_SUPPORTS-Wextra - Success
2021-12-06T06:35:12.6246746Z -- Performing Test COMPILER_SUPPORTS-Wpedantic
2021-12-06T06:35:12.7146634Z -- Performing Test COMPILER_SUPPORTS-Wpedantic - Success
2021-12-06T06:35:12.7161016Z -- Performing Test COMPILER_SUPPORTS-Wshadow
2021-12-06T06:35:12.8046167Z -- Performing Test COMPILER_SUPPORTS-Wshadow - Success
2021-12-06T06:35:12.8057472Z -- Performing Test COMPILER_SUPPORTS-Wconversion
2021-12-06T06:35:12.8942920Z -- Performing Test COMPILER_SUPPORTS-Wconversion - Success
2021-12-06T06:35:12.8958307Z -- Performing Test COMPILER_SUPPORTS_march_native
2021-12-06T06:35:12.9967108Z -- Performing Test COMPILER_SUPPORTS_march_native - Success
2021-12-06T06:35:13.0028321Z -- Looking for C++ include pthread.h
2021-12-06T06:35:13.1093430Z -- Looking for C++ include pthread.h - found
2021-12-06T06:35:13.1101273Z -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
2021-12-06T06:35:13.2193151Z -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
2021-12-06T06:35:13.2202960Z -- Looking for pthread_create in pthreads
2021-12-06T06:35:13.3016560Z -- Looking for pthread_create in pthreads - not found
2021-12-06T06:35:13.3019141Z -- Looking for pthread_create in pthread
2021-12-06T06:35:13.4019928Z -- Looking for pthread_create in pthread - found
2021-12-06T06:35:13.4036242Z -- Found Threads: TRUE  
2021-12-06T06:35:13.4169538Z -- Found Boost: $PREFIX/lib/cmake/Boost-1.74.0/BoostConfig.cmake (found suitable version "1.74.0", minimum required is "1.53.0") found components: program_options filesystem system 
2021-12-06T06:35:13.6715381Z -- Found Python: $PREFIX/bin/python3.8 (found suitable version "3.8.12", minimum required is "3") found components: Interpreter 
2021-12-06T06:35:13.9203293Z -- Check size of int
2021-12-06T06:35:14.0108042Z -- Check size of int - done
2021-12-06T06:35:14.0116475Z -- MKL: Thread Layer(GNU OpenMP) Interface(4-byte Integer)
2021-12-06T06:35:14.0326511Z -- Found OpenMP_CXX: -fopenmp (found version "4.5") 
2021-12-06T06:35:14.0334528Z -- Found OpenMP: TRUE (found version "4.5")  
2021-12-06T06:35:14.0706937Z -- Could NOT find MKL: Source the compilervars.sh or mklvars.sh scripts included with your installation of MKL. This script searches for the libraries in MKLROOT, LIBRARY_PATHS(Linux), and LIB(Windows) environment variables (missing: MKL_INCLUDE_DIR MKL_Core_LINK_LIBRARY MKL_Interface_LINK_LIBRARY MKL_ThreadLayer_LINK_LIBRARY) 
2021-12-06T06:35:14.0712390Z -- Could NOT find MKL: Source the compilervars.sh or mklvars.sh scripts included with your installation of MKL. This script searches for the libraries in MKLROOT, LIBRARY_PATHS(Linux), and LIB(Windows) environment variables (missing: MKL_INCLUDE_DIR MKL_Core_STATIC_LINK_LIBRARY MKL_Interface_STATIC_LINK_LIBRARY MKL_ThreadLayer_STATIC_LINK_LIBRARY) 
2021-12-06T06:35:14.0758001Z -- Found OpenMP_CXX: -fopenmp (found suitable version "4.5", minimum required is "4.5") 
2021-12-06T06:35:14.0766366Z -- Found OpenMP: TRUE (found suitable version "4.5", minimum required is "4.5")  
2021-12-06T06:35:14.0793834Z -- Found Eigen3: $PREFIX/share/eigen3/cmake
2021-12-06T06:35:14.0883315Z -- Found PkgConfig: /usr/bin/pkg-config (found version "0.27.1") 
2021-12-06T06:35:14.0895215Z -- Checking for module 'fftw3'
2021-12-06T06:35:14.1069260Z --   Found fftw3, version 3.3.10
2021-12-06T06:35:14.1441275Z -- Found FFTW3: $PREFIX/lib/libfftw3.so  
2021-12-06T06:35:14.1452941Z -- Looking for C++ include cassert
2021-12-06T06:35:14.2388047Z -- Looking for C++ include cassert - found
2021-12-06T06:35:14.2397046Z -- Looking for C++ include cmath
2021-12-06T06:35:14.4696876Z -- Looking for C++ include cmath - found
2021-12-06T06:35:14.4704350Z -- Looking for C++ include fstream
2021-12-06T06:35:14.8945506Z -- Looking for C++ include fstream - found
2021-12-06T06:35:14.8952159Z -- Looking for C++ include functional
2021-12-06T06:35:15.3689499Z -- Looking for C++ include functional - found
2021-12-06T06:35:15.3696124Z -- Looking for C++ include iostream
2021-12-06T06:35:15.8003655Z -- Looking for C++ include iostream - found
2021-12-06T06:35:15.8024694Z -- Looking for C++ include limits
2021-12-06T06:35:15.9130624Z -- Looking for C++ include limits - found
2021-12-06T06:35:15.9139454Z -- Looking for C++ include list
2021-12-06T06:35:16.0773972Z -- Looking for C++ include list - found
2021-12-06T06:35:16.0783230Z -- Looking for C++ include map
2021-12-06T06:35:16.4535331Z -- Looking for C++ include map - found
2021-12-06T06:35:16.4542618Z -- Looking for C++ include ostream
2021-12-06T06:35:16.8308935Z -- Looking for C++ include ostream - found
2021-12-06T06:35:16.8319958Z -- Looking for C++ include sstream
2021-12-06T06:35:17.2331443Z -- Looking for C++ include sstream - found
2021-12-06T06:35:17.2338806Z -- Looking for C++ include stack
2021-12-06T06:35:17.4168887Z -- Looking for C++ include stack - found
2021-12-06T06:35:17.4176830Z -- Looking for C++ include stdexcept
2021-12-06T06:35:17.7285102Z -- Looking for C++ include stdexcept - found
2021-12-06T06:35:17.7290303Z -- Looking for C++ include string
2021-12-06T06:35:18.0491702Z -- Looking for C++ include string - found
2021-12-06T06:35:18.0501578Z -- Looking for C++ include vector
2021-12-06T06:35:18.2351617Z -- Looking for C++ include vector - found
2021-12-06T06:35:18.2958684Z -- Found EXPAT: $PREFIX/lib/libexpat.so (found version "2.4.1") 
2021-12-06T06:35:18.3573268Z -- Found Python: $PREFIX/bin/python3.8 (found suitable version "3.8.12", minimum required is "3.5") found components: Interpreter 
2021-12-06T06:35:18.3733082Z -- Found Boost: $PREFIX/lib/cmake/Boost-1.74.0/BoostConfig.cmake (found suitable version "1.74.0", minimum required is "1.53.0") found components: program_options filesystem system regex 
2021-12-06T06:35:18.3780447Z -- Found Eigen3: $PREFIX/share/eigen3/cmake
2021-12-06T06:35:18.3781278Z -- Found VOTCA_TOOLS: 
2021-12-06T06:35:18.3919552Z -- Checking for module 'libgromacs_d'
2021-12-06T06:35:18.3949028Z --   No package 'libgromacs_d' found
2021-12-06T06:35:18.3956532Z -- Checking for module 'libgromacs'
2021-12-06T06:35:18.3984787Z --   No package 'libgromacs' found
2021-12-06T06:35:18.4017296Z -- Could NOT find GROMACS: Found unsuitable version "0", but required is at least "2018" (found GROMACS_LIBRARY-NOTFOUND)
2021-12-06T06:35:18.4030584Z -- Could NOT find GMX (missing: GMX_EXECUTABLE) 
2021-12-06T06:35:18.4037904Z -- Looking for C++ include algorithm
2021-12-06T06:35:18.9173886Z -- Looking for C++ include algorithm - found
2021-12-06T06:35:18.9188345Z -- Looking for C++ include iomanip
2021-12-06T06:35:19.3960002Z -- Looking for C++ include iomanip - found
2021-12-06T06:35:19.3969241Z -- Looking for C++ include numeric
2021-12-06T06:35:19.5260409Z -- Looking for C++ include numeric - found
2021-12-06T06:35:19.5269040Z -- Looking for C++ include cstdlib
2021-12-06T06:35:19.6272586Z -- Looking for C++ include cstdlib - found
2021-12-06T06:35:20.3111442Z -- HDF5 CXX compiler wrapper is unable to compile a minimal HDF5 program.
2021-12-06T06:35:20.3257647Z -- Found HDF5: $PREFIX/lib/libhdf5_cpp.so;$PREFIX/lib/libhdf5.so (found version "1.10.6") found components: CXX 
2021-12-06T06:35:20.3809485Z -- Could NOT find LMP (missing: LMP_EXECUTABLE) 
2021-12-06T06:35:20.4141311Z -- Found Python: $PREFIX/bin/python3.8 (found suitable version "3.8.12", minimum required is "3") found components: Interpreter 
2021-12-06T06:35:20.5734175Z -- Found python module pytest
2021-12-06T06:35:20.5910268Z -- Found libecpint: $PREFIX/lib/cmake/ecpint
2021-12-06T06:35:21.2643873Z -- HDF5 CXX compiler wrapper is unable to compile a minimal HDF5 program.
2021-12-06T06:35:21.2750958Z -- Found HDF5: $PREFIX/lib/libhdf5_cpp.so;$PREFIX/lib/libhdf5.so (found suitable version "1.10.6", minimum required is "1.8") found components: CXX 
2021-12-06T06:35:21.2759013Z CMake Warning at xtp/CMakeLists.txt:57 (message):
2021-12-06T06:35:21.2759710Z   HDF5 will be used such that it is compatible with version 1.8.
2021-12-06T06:35:21.2759992Z 
2021-12-06T06:35:21.2760198Z 
2021-12-06T06:35:21.2769504Z -- Performing Test COMPILER_SUPPORTS_WNO_DEPRECATED_COPY
2021-12-06T06:35:21.3731045Z -- Performing Test COMPILER_SUPPORTS_WNO_DEPRECATED_COPY - Success
2021-12-06T06:35:21.3919214Z -- Found Boost: $PREFIX/lib/cmake/Boost-1.74.0/BoostConfig.cmake (found suitable version "1.74.0", minimum required is "1.57.0") found components: program_options filesystem system timer 
2021-12-06T06:35:21.3948088Z -- Found Eigen3: $PREFIX/share/eigen3/cmake
2021-12-06T06:35:21.4467972Z -- Found VOTCA_TOOLS: 
2021-12-06T06:35:21.4469854Z -- Found VOTCA_CSG: 
2021-12-06T06:35:21.4980814Z Traceback (most recent call last):
2021-12-06T06:35:21.4981646Z   File "<string>", line 1, in <module>
2021-12-06T06:35:21.4982703Z ModuleNotFoundError: No module named 'lxml'
2021-12-06T06:35:21.5208783Z Traceback (most recent call last):
2021-12-06T06:35:21.5209622Z   File "<string>", line 1, in <module>
2021-12-06T06:35:21.5210972Z ModuleNotFoundError: No module named 'h5py'
2021-12-06T06:35:21.5608968Z -- Found python module rdkit
2021-12-06T06:35:21.6850529Z -- Could NOT find JUPYTER_NBCONVERT: Found unsuitable version "0.0", but required is at least "5.4.1" (found /opt/conda/bin/jupyter)
2021-12-06T06:35:21.6885172Z -- Could NOT find SPHINX: Found unsuitable version "0.0", but required is at least "2.1" (found SPHINX_EXECUTABLE-NOTFOUND)
2021-12-06T06:35:21.6937753Z -- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE) 
2021-12-06T06:35:21.6950844Z -- Could NOT find SPHINX (missing: SPHINX_EXECUTABLE) 
2021-12-06T06:35:21.7019924Z -- Could NOT find CLANG_FORMAT: Found unsuitable version "0.0", but required is at least "11" (found CLANG_FORMAT_EXECUTABLE-NOTFOUND)
2021-12-06T06:35:21.7169847Z -- The following features have been enabled:
2021-12-06T06:35:21.7170774Z 
2021-12-06T06:35:21.7171265Z  * BUILD_XTP, Build xtp module
2021-12-06T06:35:21.7172111Z  * INJECT_MARCH_NATIVE, Add -march=native to CXX flags
2021-12-06T06:35:21.7172995Z  * BUILD_MANPAGES, Build manpages (disable for cross-compile)
2021-12-06T06:35:21.7181423Z  * INSTALL_RC_FILES, Install votca rc files (VOTCARC.bash etc.)
2021-12-06T06:35:21.7181960Z 
2021-12-06T06:35:21.7200262Z -- The following RECOMMENDED packages have been found:
2021-12-06T06:35:21.7200855Z 
2021-12-06T06:35:21.7201250Z  * FFTW3
2021-12-06T06:35:21.7201672Z    Enables FFT functionality
2021-12-06T06:35:21.7202080Z  * PkgConfig
2021-12-06T06:35:21.7202503Z    Used to detect fftw package
2021-12-06T06:35:21.7202940Z    Used to detect gromacs package
2021-12-06T06:35:21.7203241Z 
2021-12-06T06:35:21.7203902Z -- The following REQUIRED packages have been found:
2021-12-06T06:35:21.7204302Z 
2021-12-06T06:35:21.7204697Z  * OpenMP (required version >= 4.5)
2021-12-06T06:35:21.7205184Z    Used for thread parallelization in tools
2021-12-06T06:35:21.7205613Z  * EXPAT
2021-12-06T06:35:21.7206016Z    Used for XML parsing
2021-12-06T06:35:21.7206459Z  * boost_regex (required version == 1.74.0)
2021-12-06T06:35:21.7206897Z  * Threads
2021-12-06T06:35:21.7207303Z    Used for thread parallelization
2021-12-06T06:35:21.7207725Z  * Libxc
2021-12-06T06:35:21.7208331Z    Provides exchange-correclation functionals
2021-12-06T06:35:21.7208884Z  * libint2
2021-12-06T06:35:21.7209303Z    Calculates Gaussian integrals
2021-12-06T06:35:21.7209706Z  * ecpint
2021-12-06T06:35:21.7210161Z    Calculates Gaussian integrals over pseudo potentials
2021-12-06T06:35:21.7210637Z  * HDF5 (required version >= 1.8)
2021-12-06T06:35:21.7211091Z    Used to read h5md data files
2021-12-06T06:35:21.7211529Z    Used to read/write HDF5 data files
2021-12-06T06:35:21.7212139Z  * boost_program_options (required version == 1.74.0)
2021-12-06T06:35:21.7212847Z  * boost_filesystem (required version == 1.74.0)
2021-12-06T06:35:21.7213611Z  * boost_system (required version == 1.74.0)
2021-12-06T06:35:21.7214150Z  * boost_chrono (required version == 1.74.0)
2021-12-06T06:35:21.7214628Z  * boost_headers (required version == 1.74.0)
2021-12-06T06:35:21.7215122Z  * boost_timer (required version == 1.74.0)
2021-12-06T06:35:21.7215583Z  * Boost (required version >= 1.57.0)
2021-12-06T06:35:21.7216037Z    Extended C++ libraries
2021-12-06T06:35:21.7216471Z  * Eigen3 (required version >= 3.3.0)
2021-12-06T06:35:21.7216935Z    C++ vector data structures
2021-12-06T06:35:21.7217363Z  * Python (required version >= 3)
2021-12-06T06:35:21.7217829Z    Used to build documentation files
2021-12-06T06:35:21.7218279Z    Used for csg scripts
2021-12-06T06:35:21.7218709Z    Used to build documentation files
2021-12-06T06:35:21.7219174Z    Used to build documentation files
2021-12-06T06:35:21.7219619Z    Used to build documentation files
2021-12-06T06:35:21.7219925Z 
2021-12-06T06:35:21.7220627Z -- The following features have been disabled:
2021-12-06T06:35:21.7220977Z 
2021-12-06T06:35:21.7221396Z  * ENABLE_TESTING, Enable unit tests
2021-12-06T06:35:21.7221870Z  * ENABLE_RPATH_INJECT, Inject rpath into executables
2021-12-06T06:35:21.7222412Z  * BUILD_OWN_GROMACS, Build an internal version of gromacs
2021-12-06T06:35:21.7222952Z  * BUILD_OWN_LIBINT, Build an internal version of libint
2021-12-06T06:35:21.7223570Z  * BUILD_CSGAPPS, Install extra app using csg
2021-12-06T06:35:21.7224267Z  * ENABLE_REGRESSION_TESTING, Enable regression tests
2021-12-06T06:35:21.7224748Z  * USE_CUDA, Use CUDA for GPU acceleration
2021-12-06T06:35:21.7225133Z 
2021-12-06T06:35:21.7225801Z -- The following OPTIONAL packages have not been found:
2021-12-06T06:35:21.7226171Z 
2021-12-06T06:35:21.7226537Z  * MKL
2021-12-06T06:35:21.7226967Z    Enables accelerated performance with MKL
2021-12-06T06:35:21.7227445Z  * JUPYTER_NBCONVERT (required version >= 5.4.1)
2021-12-06T06:35:21.7227938Z  * SPHINX (required version >= 2.1)
2021-12-06T06:35:21.7228341Z  * Doxygen
2021-12-06T06:35:21.7228765Z    Needed to build API documentation
2021-12-06T06:35:21.7229375Z  * CLANG_FORMAT (required version >= 11)
2021-12-06T06:35:21.7229688Z 
2021-12-06T06:35:21.7230352Z -- The following RECOMMENDED packages have not been found:
2021-12-06T06:35:21.7230724Z 
2021-12-06T06:35:21.7231140Z  * GROMACS (required version >= 2018)
2021-12-06T06:35:21.7231620Z    Used to read/write gromacs data files
2021-12-06T06:35:21.7231906Z 
2021-12-06T06:35:21.7464959Z -- Configuring done
2021-12-06T06:35:22.0280133Z -- Generating done
```

**Comment by jan-janssen** [comment]
I opened a separate pull request for ` libecpint` https://github.com/conda-forge/staged-recipes/pull/17177

**Comment by jan-janssen** [comment]
https://github.com/conda-forge/staged-recipes/pull/17176/commits/aee851d430de2bfbfc74b007c65fe97428b0b37c - worked for Python > 3.9. Python 3.7 failed in https://github.com/conda-forge/staged-recipes/pull/17176/commits/b4ae98af5d548f28edbf16a781c7e5ef57249dea so I am currently trying to limit the build to Python 3.8 and 3.9

**Comment by jan-janssen** [comment]
I also tried building for macOS but there was a dependency issue in https://github.com/conda-forge/staged-recipes/pull/17176/commits/3fc4e62bc75aa632fe8641911a27b507c58c5813 . I am going to take another look at this once the pull request is merged.

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/votca/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+mkdir build
+cd build
+cmake -DBUILD_XTP=ON -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_INSTALL_LIBDIR=lib -DINJECT_MARCH_NATIVE=OFF ..
+make  # -j${NUM_CPUS}
+make install
```
Comment (ocefpaf) [inline recipes/votca/build.sh:6]: Is there a make check?

Comment (jan-janssen) [reply recipes/votca/build.sh:6]: I do not think so but I can try.

Comment (jan-janssen) [reply recipes/votca/build.sh:7]:
```
make: *** No rule to make target 'check'.  Stop.
```

**Comment by jan-janssen** [comment]
@ocefpaf The `make check` part did not work as it is not implemented. Can you take another look at the recipe?

**Pull request merged**